### PR TITLE
Issues/3250 site preview tracking

### LINKED ISSUE #3250: Reader: Add tracking metrics to the site preview.
Let's if its accessed, and if its accessed from a list or from a detail. Let's also see how often the site's viewed via the in app browser. 
cc @sendhil

### PR DESCRIPTION
**Pull request by aerych**
Closes #3250

### CODE REVIEW

**Comment by aerych** [comment]
cc @sendhil Would you be up for a quick review?

File: WordPress/Classes/ViewRelated/Reader/ReaderPostsViewController.m [hunk 1]
```diff
@@ -1032,7 +1034,8 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
     detailController.readerViewStyle = self.readerViewStyle;
     [self.navigationController pushViewController:detailController animated:YES];
 
-    [WPAnalytics track:WPAnalyticsStatReaderOpenedArticle];
+    NSString *detailType = (self.readerViewStyle == ReaderViewStyleNormal) ? ReaderDetailTypeNormal : ReaderDetailTypePreviewSite;
+    [WPAnalytics track:WPAnalyticsStatReaderOpenedArticle withProperties:@{@"post-detail-type":detailType}];
```
Comment (sendhil) [inline WordPress/Classes/ViewRelated/Reader/ReaderPostsViewController.m]: Seems odd to have two out of three of these strings as constants. Why not the third?

Comment (aerych) [inline WordPress/Classes/ViewRelated/Reader/ReaderPostsViewController.m]: Ah! Good point!

**Comment by sendhil** [comment]
:shipit:

**Pull request merged**